src/components/Dropdown/MultiSelectDropdown.jsx

```diff
@@ -86,15 +86,13 @@ export class MultiSelectDropdown extends Component {
 
   render() {
     const { value, placeholder, textField, ...rest } = this.props;
-
-    const { data, isOpen, inputValue } = this.state;
-
+    const { data, isOpen, inputValue = '' } = this.state;
```

Comment:
Hvorfor gjør du dette? Vi har jo kontroll på hva default state skal være, burde i såfall endres der.

Reply:
liker du ikke "default" value på destructor?

Reply:
Tja, siden input value i teorien kan være `null` vil ikke default verdient kicke inn på den, men bare undefined. I tillegg ser jeg ikke behovet for default value siden vi styrer state selv og har kontroll på hva den kan være hele tiden. Foreslår at den isteden er empty string som default i state.